scss/civicrm/common/_buttons.scss

```diff
@@ -408,3 +408,15 @@ button {
     }
   }
 }
+
+// We style only plain buttons which are <div>s/<button>s with `crm-button` class
+// and do *not* have `crm-button-type-*` class. <span>s should not be touched.
+// We do *not* extend to `%btn-civi` because we want the height to be preserved.
+// Because of that we need to additionally discard the text shadow though.
+div,
+button {
+  &.crm-button:not([class*='crm-button-type']) {
```

Comment:
Shouldn't this be 
```
&.crm-button:not([class*='crm-button-type-']) {
```
rather for better understanding?

Reply:
Done